### PR TITLE
Gameini updates.

### PR DESCRIPTION
**Pull request by Linktothepast**
Update/addition of  Madden NFL 06 (gc), Frogger Beyond (gc), DONKEY
KONGA (gc), Madden NFL 2005 (gc), NBA STREET Vol.2 (gc), Spider-Man
(gc), Star Wars: Rogue Leader (gc), NO MORE HEROES (wii), Super Smash
Bros. Brawl (wii), The Simpsons Game (wii), Suzumiya Haruhi no Heiretsu
(wii).

Fixes issues 7978, 7894, 7883.

### CODE REVIEW

**Comment by Linktothepast** [comment]
Btw this sets efb to ram as default option for Super Smash Bros. Brawl, which cuts framerate about to half of what it was. It is needed to take ingame pics and play nes masterpieces. Nes masterpiecies also need safetexture cache set to safe, but that would hamper performance even more so i didn't enable it. I thought of using texture cache in the middle which is good enough for most cases but then it would hamper those who had stc set to safe from playing them properly.

**Comment by JosJuice** [comment]
Why enable EFB to RAM in SSBB? Playing NES games is something that's done so rarely that it isn't much of an inconvenience to change a setting when you want to do it, and SSBB's screenshot tool takes images of worse quality and saves them in a more obscure way compared to Dolphin's built-in screenshot tool. I'd say that performance in the entire game is more important than being able to do these two things easily.

File: Data/Sys/GameSettings/GSWD64.ini [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# GSWD64 - Star Wars: Rogue Leader
+
+[Core]
+# Values set here will override the main dolphin settings.
+MMU = 1
+FastDiscSpeed = 1
```
Comment (magumagu) [inline Data/Sys/GameSettings/GSWD64.ini]: PR #1778 makes this line unnecessary.

Comment (Linktothepast) [inline Data/Sys/GameSettings/GSWD64.ini]: I can change it but i think pr #1778 should be merged first before doing that or in the meanwhile the game won't work if the opposite happens. 
@JosJuice would a note suffice then? I would like a few more opinions though, i wrote about it at the forum too and only jmc answered, most people didn't seem to care to answer.

Comment (magumagu) [inline Data/Sys/GameSettings/GSWD64.ini]: If this gets merged first, it's okay to leave as-is; just wanted to make sure you were aware of the interaction.

Comment (magumagu) [inline Data/Sys/GameSettings/GSWD64.ini]: Oh, #1778 just got merged.

**Comment by Fuzzik** [comment]
I've gotta agree with JosJuice about EFB to RAM. Even with all of the neat fixes merged to make the Masterpieces work, I don't think defaulting them on is worth halving the FPS. IMO I'd just add in a note regarding that it's needed for those two features.

**Comment by Linktothepast** [comment]
Ok done, i removed FastDiscSpeed from Rogue Leader and efb to ram from Super Smash Bros. Brawl, put a long note about it instead.

**Comment by JMC47** [comment]
These look good to me after a quick runthrough.  I would have approved of Brawl regardless of the decision, it's such a tough one.

**Comment by ZephyrSurfer** [comment]
Wait up on this I'm about to make a pull request but the game will still require Virtual XFB set.

**Comment by Linktothepast** [comment]
Oh no, i was about to push some new inis. Guess i will have to make a new pr then, sorry ZephyrSurfer, perhaps i will include changes for the zfreeze pr as well.

**Comment by cdoublejj** [comment]
Hey would I be asking too much for someone to checkout the ini for Rabbids Go Home?

https://forums.dolphin-emu.org/Thread-rabbids-go-home-issues?pid=351336#pid351336

**Comment by Linktothepast** [comment]
That game has some weird performance issues that make me not want to enforce a setting. It needs xfb but anything higher than 1x ir for some reason kills performance, so i would have to choose between killing performance or killing quality by default.

**Comment by cdoublejj** [comment]
real xfb requires 1x feed back any ways. my understanding is that it needs real xfb and 1x IR. though for me real xfb made no difference i wasn't sure if it was the game or a dolphin bug. do the intro videos play corrupted and distorted for you as well?

**Comment by Linktothepast** [comment]
With real xfb and safe texture cache in the middle the game plays fine here, also efb to ram is needed for some ingame functions like taking photos but that murders the framerate. With virtual xfb and any ir higher than 1x performance gets slaughtered too.
So to answer your question with real xfb and stc in the middle videos are perfectly fine here.

**Comment by cdoublejj** [comment]
Well i think i figured out why these settings were not working for me scaled EFB is enabled globally and with it off the game started working fine. it's still got crackly audio but, much better. is it possible to disable scaled EFB for rabbids and go home and set the setting you mention in the game ini?
realxfb set to ram and texture cache half way with scaled EFB disabled? other wise without these settings the game is broken, at least in the videos.

now i just need to test and see if AA and atniscropic can applied but, i wouldn't get my hopes up.

**Pull request merged**